### PR TITLE
Add backend service implementation for geofence and add geofence to b…

### PR DESCRIPTION
**Pull request by coderkalyan**
Implement backend service for geofence

### CODE REVIEW

File: tools/generate_from_protos.sh [hunk 1]
```diff
@@ -24,7 +24,7 @@ command -v ${protoc_binary} && command -v ${protoc_grpc_binary} || {
     exit 1
 }
 
-plugin_list="action calibration gimbal camera core info mission offboard param telemetry"
+plugin_list="action calibration gimbal camera core info mission offboard param telemetry geofence"
```
Comment (JonasVautherin) [inline tools/generate_from_protos.sh]:
```suggestion
plugin_list="action calibration geofence gimbal camera core info mission offboard param telemetry"
```

Comment (coderkalyan) [reply tools/generate_from_protos.sh]: Just ran style checker. I tried to keep styling the same - probably just indentation/editor specific things :wink:

**Comment by coderkalyan** [comment]
@JonasVautherin I removed the src/backend/proto from the request. Just curious - where are the proto files kept these days? Anyway, please take a look and let me know if there's anything else I need to do!

**Comment by JonasVautherin** [comment]
> where are the proto files kept these days?

They are now at the [root of the repo](https://github.com/mavlink/mavsdk), in `proto/` :blush:.

**Comment by coderkalyan** [comment]
No problem! Happy to help, and thanks for merging

**Pull request merged**